data/campaigns/Secrets_of_the_Ancients/utils/zombie-utils.cfg

```diff
@@ -39,7 +39,7 @@
             name=zombies
             mode=merge
             [split]
-                list= _ "(Bat),
+                list= "(Bat),
```

Comment:
This looks like it's just terrible code. This is not a good way to implement this, at all. I would say this code just needs a major overhaul. In particular, all the unit names in this list are already translated in the core game data - why does SotA need to have its own version of each one?

I've opened #5170 to track this as a separate issue. In the meantime, the removal of these two translation marks should be reverted.

Reply:
reverted removal of translation marks in ad68ef0